postgresql/resource_postgresql_grant_role.go

```diff
@@ -0,0 +1,222 @@
+package postgresql
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	"github.com/lib/pq"
+)
+
+const (
+	// This returns the role membership for role, grant_role
+	getGrantRoleQuery = `
+SELECT pg_get_userbyid(member) as role, pg_get_userbyid(roleid) as grant_role, admin_option
+FROM pg_auth_members
+WHERE pg_get_userbyid(member) = $1
+AND pg_get_userbyid(roleid) = $2;
```

Comment:
Nit:

```suggestion
SELECT
  pg_get_userbyid(member) as role,
  pg_get_userbyid(roleid) as grant_role,
  admin_option
FROM
  pg_auth_members
WHERE
  pg_get_userbyid(member) = $1 AND
  pg_get_userbyid(roleid) = $2;
```

(feel free to say no)